src/rprblender/nodes/blender_nodes.py

```diff
@@ -972,6 +972,39 @@ def export_hybrid(self):
         return shader2
 
 
+class ShaderNodeObjectInfo(NodeParser):
+    
+    def export(self):
+        if self.socket_out.name == 'Location':
+            if self.object:
+                return self.node_item(self.object.location)
```

Comment:
should be: return self.node_item(tuple(self.object.location))

Reply:
Clearly I didn't test all the inputs.  However I could claim that maybe node_item should take any iterable with the right length.  What do you think?

Reply:
Not just tuple.